buildpipeline/Core-Setup-FreeBSD-BT.json

```diff
@@ -200,6 +200,9 @@
     "PB_PackageVersionPropsUrl": {
       "value": ""
     },
+    "PB_SkipTests": {
+      "value": "true"
```

Comment:
Why set this and a manual build argument?  Is something anticipating an output if this isn't set to false?

Reply:
not needed. Here is update run. https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/results?buildId=2199610&view=logs